client/account-recovery/account-recovery-root/index.jsx

```diff
@@ -35,13 +39,35 @@ const getPageInfo = ( translate, step ) => {
 			trackerTitle: 'Account Recovery > Reset Password',
 			documentHeadTitle: translate( 'Reset Password ‹ Account Recovery' ),
 		},
-
+		[ STEPS.RESET_PASSWORD_EMAIL ]: {
+			trackerTitle: 'Account Recovery > Reset Password Email',
+			documentHeadTitle: translate( 'Reset Password < Email' ),
```

Comment:
As mentioned in slack the `73 existing translations will be lost with this change` is wrong. 
However, if the title is in a breadcrumb format "page a | option 1" then it is okay to concat `translate( "page a" ) + < + translate ("option 1")`  like you mentioned.